lib/streamlit/elements/image.py

```diff
@@ -114,12 +121,23 @@ def image(
             Image width. None means use the image width,
             but do not exceed the width of the column.
             Should be set for SVG images, as they have no default image width.
+        use_container_width : bool
```

Comment:
nit: Perhaps we should move the parameter doc to the bottom to match the same argument sequence as in the function definition. What do you think?

Reply:
Sounds reasonable! Will update